api/api.yaml

```diff
@@ -166,9 +166,12 @@ definitions:
     description: >
       Can be a field (Gender), or a field and selected value (Gender = female).
```

Comment:
Since we're no longer returning "fields", making some more `field` -> `facet` suggestions.

Can represent a facet (Gender), or a facet and selected value (Gender = female).